ICSharpCode.Decompiler/CSharp/CSharpDecompiler.cs

```diff
@@ -873,6 +873,14 @@ EntityDeclaration DoDecompile(FieldDefinition fieldDefinition, IField field, ITy
 				RemoveAttribute(fixedFieldDecl, fixedBufferAttributeTypeName);
 				return fixedFieldDecl;
 			}
+			var cecilField = typeSystem.GetCecil(field) as FieldDefinition;
```

Comment:
No need to use GetCecil here... the first parameter already is the correct Cecil FieldDefinition.

Reply:
Great point -- tried to update quickly and missed that.
Fixed now.